src/west/commands/project.py

```diff
@@ -46,7 +46,7 @@ def do_run(self, args, user_args):
         for project in _all_projects(args):
             log.inf('{:14}  {:18}  {:13}  {}  {}'.format(
                 project.name,
-                os.path.join(project.path, ''),  # Add final '/' if missing
+                project.path,
```

Comment:
This was just a small nice-to-have, to make it clear in the output that it's a directory. You can remove it if you think it helps though.

Note that `foo//bar` works the same as `foo/bar`, so redundant slashes won't break anything when opening files. This is POSIX behavior and built right into the kernel.

Reply:
Personally I have no special feelings on this, although the version without `/` (@mbolivar suggestion) at the end is more in-line what you will see with `ls` or `dir` in windows.

Although in both windows you'll see `<DIR>` and in linux, you most often has coloring of the output, which help to identify folders when printed on screen (or `ls -l` shows a `d---------` at start).
Such identification we don't have in `west list`.

But what I think is most important is that we stay consistent.
Either a trailing `/` at the end everywhere we print folders, or not at all.

Reply:
> In windows you'll see <DIR> and in linux, you most often has coloring of the output, which help > to identify folders when printed on screen (or ls -l shows a d--------- at start).
> Such identification we don't have in west list.
> But what I think is most important is that we stay consistent.
> Either a trailing `/` at the end everywhere we print folders, or not at all.

I played around with the "dir" options on windows, and I couldn't find any one that appends a trailing '\' to the output, which is what we get with "west list" right now. So this patch makes west consistent with both `ls` on POSIX operating systems and `dir` on Windows. I'm going to keep it as-is since we all seem OK with that.



As @tejlmand points out, the default `dir` output prefixes directories with <DIR>:

```
C:\Users\mbolivar\west>dir
 Volume in drive C is Windows
 Volume Serial Number is 6072-12F7

 Directory of C:\Users\mbolivar\west

11/01/2018  05:50 AM    <DIR>          .
11/01/2018  05:50 AM    <DIR>          ..
11/01/2018  05:49 AM                81 .gitignore
11/01/2018  05:50 AM    <DIR>          .pytest_cache
11/01/2018  05:49 AM               728 .shippable.yml
11/01/2018  05:49 AM            11,558 LICENSE
11/01/2018  05:49 AM             4,812 README.rst
11/01/2018  05:49 AM    <DIR>          scripts
11/01/2018  05:49 AM             1,348 setup.py
11/01/2018  05:49 AM    <DIR>          src
11/01/2018  05:49 AM    <DIR>          tests
11/01/2018  05:49 AM                36 tests_requirements.txt
               6 File(s)         18,563 bytes
               6 Dir(s)  78,789,935,104 bytes free
```

The "wide" option surrounds directories with square brackets:

```
C:\Users\mbolivar\west>dir /w
 Volume in drive C is Windows
 Volume Serial Number is 6072-12F7

 Directory of C:\Users\mbolivar\west

[.]                      [..]                     .gitignore               [.pytest_cache]          .shippable.yml           LICENSE
README.rst               [scripts]                setup.py                 [src]                    [tests]                  tests_requirements.txt
               6 File(s)         18,563 bytes
               6 Dir(s)  78,789,935,104 bytes free
```

The "bare" format just prints the names:

```
C:\Users\mbolivar\west>dir /b
.gitignore
.pytest_cache
.shippable.yml
LICENSE
README.rst
scripts
setup.py
src
tests
tests_requirements.txt
```

Nothing appends a backslash though.